instrumentation/ratpack/ratpack-1.7/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("io.ratpack")
+    module.set("ratpack-core")
+    versions.set("[1.7.0,)")
+  }
+}
```

Comment:
could you add
```
  fail {
    group.set("io.ratpack")
    module.set("ratpack-core")
    versions.set("[1.0,1.7)")
  }
```
to verify that the instrumentation does not apply to earlier versions